core/trino-main/src/test/java/io/trino/cost/TestStatsCalculator.java

```diff
@@ -32,11 +34,12 @@
 
 public class TestStatsCalculator
 {
-    private final LocalQueryRunner queryRunner;
+    private LocalQueryRunner queryRunner;
 
-    public TestStatsCalculator()
+    @BeforeClass
+    public void setUp()
     {
-        this.queryRunner = LocalQueryRunner.create(testSessionBuilder()
+        queryRunner = LocalQueryRunner.create(testSessionBuilder()
```

Comment:
the `This is really bad idea.` in commit messages is not really helpful -would drop.

Reply:
i don't want to explain this in full in every commit, but some explanation may be due, so this just conveys there is something important to it